mitosheet/mitosheet/pro/updates/delete_steps_after_idx.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+"""
+Rolls back/forward to a specific step, by index
+"""
+
+from mitosheet.types import StepsManagerType
+
+
+DELETE_STEPS_AFTER_IDX_UPDATE_EVENT = 'delete_steps_after_idx_update'
```

Comment:
Can we rename this to `undo_to_step_index_update`? I think it's more clear -- delete is a new and bit confusing word.